### PR TITLE
feature(plugins): Implemented API interface for plugins

### PR DESCRIPTION
**Pull request by Daboo1998**

### CODE REVIEW

**Review by PAKovalev — APPROVED** [review]

**Review by aminbksi — APPROVED** [review]

File: packages/app/src/scenes/collaboration/pages/CollaborationPluginPage/CollaborationPluginPage.tsx [hunk 1]
```diff
@@ -116,12 +83,7 @@ const CollaborationPluginPage: FC<PropsInterface> = ({pluginLoader}) => {
                 theme={theme}
                 isSpaceAdmin={space.isAdmin}
                 spaceId={space.id}
-                init={init}
-                reload={async () => {
-                  await pluginLoader.sharedState.reload();
-                }}
-                sharedState={pluginLoader.sharedState.data}
-                setSharedState={setSharedState}
+                api={attributesManager.getAPI(worldStore.worldId, space.id)}
```
[the remark this answers is not in the document]
Comment (dmitry-yudakov) [reply packages/app/src/scenes/collaboration/pages/CollaborationPluginPage/CollaborationPluginPage.tsx:86]: This way the api instance is recreated on every render and will mess up the deps lists in different hooks in user's code.

**Review by dmitry-yudakov — COMMENTED** [review]
It has one typo in package.json and potential for rendering busy loop. The interface is ok for now although I don't like the naming.

File: packages/sdk/package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "start": "tsc -w"
   },
   "dependencies": {
-    "@momentum-xyz/ui-kit": "^0.1.1",
+    "@momentum-xyz/ui-kit": "^0.1.w",
```
[the remark this answers is not in the document]
Comment (Daboo1998) [reply packages/sdk/package.json:19]: I won't deploy yet, but version is ok to bump already I think

File: packages/app/src/stores/MainStore/models/PluginsStore/PluginsStore.ts [hunk 3]
```diff
@@ -41,17 +46,22 @@ const PluginsStore = types
         self.pluginOptionsRequest.send(api.pluginsRepository.getPluginsOptions, {plugin_uuids})
       ]);
 
-      const plugins = Object.entries(
-        pluginsMetadata as GetPluginsMetadataResponse
-      ).map<PluginInterface>(([plugin_uuid, metadata]) => {
-        const options = (pluginsOptions as GetPluginsOptionsResponse)[plugin_uuid];
+      const plugins = Object.entries(pluginsMetadata as GetPluginsMetadataResponse)
+        .map<PluginInterface>(([plugin_uuid, metadata]) => {
```
[the remark this answers is not in the document]
Comment (Daboo1998) [reply packages/app/src/stores/MainStore/models/PluginsStore/PluginsStore.ts:50]: I wanted it be more in line what BE returns, then later you set it to pluginId

**Pull request merged**